### ISSUE TITLE
Multiple relation tabs including filters result in error

### ISSUE THREAD

**Issue by Simon-Edelmann** [eminiarts/nova-tabs#13]
Hi,
I've just found your package. Great work!

It seems that there is a problem with filters, when I have two tabs with different relations:
```
(new TabsPanel('Project Details', [
  Tabs::make()
    ->addTab('Applications', HasMany::make('Applications'))
    ->addTab('Information', [
      // other fields
    ])
    ->addTab('Instruments', HasMany::make('Instruments')),
  ]))->withToolbar(),
];
```
My Nova resource 'Application' has a filter, 'Instrument' does not have one. After loading the TabsPanel I get this error. Also the filter button is not displayed.
<img width="578" alt="bildschirmfoto 2019-01-26 um 13 16 08" src="https://user-images.githubusercontent.com/2821076/51787002-94508400-216c-11e9-9154-98e69dc0164b.png">
I can get the filter button back when I refresh the page (F5) but my filter is displayed on both relationship tabs then.

**Comment by dkulyk** [comment]
What is Nova version?

**Comment by bajramemini** [comment]
Should be fixed in [1.0.0](https://github.com/eminiarts/nova-tabs/releases/tag/1.0.0)

Let me know if you have further issues.

**Comment by Simon-Edelmann** [comment]
Thanks, it's working now! 
@bajramemini @dkulyk Great work!
